spec/pandoc_parser_spec.rb

```diff
@@ -1,63 +1,35 @@
 require 'spec_helper'
+require 'pandoc-ruby'
 
-describe Jekyll::Converters::Markdown::PandocParser do
+describe Jekyll::Converters::Markdown::PandocProcessor do
 
-  let(:config) {{ 'markdown' => 'pandoc',
-                  'pandoc' => { 'extensions' => ["smart", "mathjax"] }}}
-  let(:markdown) { "## This is a heading" }
-  let(:html) { "<h2 id=\"this-is-a-heading\">This is a heading</h2>\n" }
 
-  describe "pandoc-ruby" do
-    subject { PandocRuby }
-
-    it "should convert markdown" do
-      subject.new(markdown).to_html.should eq(html)
+  describe 'pandoc' do
+    config = {'markdown' => 'pandoc'}
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.